play/src/front/Phaser/Game/MapEditor/MapEditorModeManager.ts

```diff
@@ -185,18 +206,28 @@ export class MapEditorModeManager {
         }
     }
 
-    public isActive(): boolean {
-        return this.active;
+    /**
+     * Update local map with missing commands given from the map-storage on RoomJoinedEvent. This commands
+     * are applied locally and are not being send further.
+     * @param commands Commands to apply in order to make sure the local map is in sync with the map-storage
+     */
+    public updateMapToNewest(commands: EditMapCommandMessage[]) {
+        for (const command of commands) {
+            for (const key of Object.keys(this.editorTools)) {
+                this.editorTools[key as EditorToolName].handleIncomingCommandMessage(command);
+            }
```

Comment:
You can probably simplify this to:

```suggestion
            for (const tool of Object.values(this.editorTools)) {
                tool.handleIncomingCommandMessage(command);
            }
```

(same thing for the rest of the file)